README.md

```diff
@@ -7,12 +7,17 @@ Its basically an application making engine written in **C++**<br/>
 This can also be used an powerful **GameEngine**!<br/>
 
 ## :wrench: Project Setup :wrench:
-- This Project uses `Premake` as its build system (`CMake` will be added later)<br/>
+- This Project uses `Premake` as its build system (`CMake` support now available!)<br/>
```

Comment:
Please Make this line as 
 This Project uses `Premake` and `CMake` as its build system.